File: example/vwo/php/sample-VWO.php

FeatureFlag::start("your_env_id", "your_api_key");
$visitor = FeatureFlag::newVisitor("your_visitor_id")->build();
$visitor->updateContext("isVip", true);
$visitor->fetchFlags();

$flag = $visitor->getFeatureVariableValue($campaignKey, 'VWO-flag-php', 'user_123', $options);

File: example/vwo/java/sample-VWO.java
Visitor visitor = featureFlag.newVisitor("visitor_unique_id").build();
visitor.updateContext("isVip", true);
visitor.fetchFlags().whenComplete((instance, error) -> {
    
    Flag<String> btnColorFlag = visitor.getFeatureVariableValue(campaignKey, "VWO-flag-java", "user_123", options);

});

